docs/content/en/docs/getting-started/production-environment/gs-mgmt-cluster.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Create production cluster"
+weight: 40
+---
+
+EKS Anywhere supports a vSphere provider for production grade EKS Anywhere deployments.
+EKS Anywhere allows you to provision and manage Amazon EKS on your own infrastructure.
+
+This document walks you through setting up EKS Anywhere in a way that:
+
+* Deploys a management cluster on your vSphere environment
+* Deploys one or more workload clusters from the management cluster
+* Keeps the management cluster in place so you can use it to later upgrade and delete workload clusters
+
+% alert title="Important" color="primary" %
+
+In the initial release, EKS Anywhere clusters functioned as both workload and management clusters.
+CAPI functions moved back and forth before between temporary bootstrap clusters running on your administrative machine and the combined cluster.
+
+Separating management features into a separate, persistant management cluster
+provides a cleaner model for managing the lifecycle of workload clusters (to create, upgrade, and delete clusters), while workload clusters run user applications.
+This approach also allows for segmentation of different cluster types.
+
+% /alert %
+
+## Prerequisite Checklist
+
+EKS Anywhere needs to be run on an administrative machine that has certain [machine
+requirements]({{< relref "../install" >}}).
+An EKS Anywhere deployment will also require the availability of certain
+[resources from your VMware vSphere deployment]({{< relref "/docs/reference/vsphere/vsphere-prereq/_index.md" >}}).
+
+## Steps
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Generate a management cluster config:
+   ```bash
+   CLUSTER_NAME=sigb
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME \
+      --provider vsphere > eksa-mgmt-cluster.yaml
+   ```
+
+1. Modify the management cluster config (`eksa-mgmt-cluster.yaml`):
+
+   Create at least three control plane nodes and three worker nodes for a production cluster, to provide high availability and rolling upgrades. The following example contains that number, plus three separate nodes for etcd:
+
+   % alert title="Note" color="primary" %
+   The config file shown below also notes which values should be changed if you want to reuse the file to create workload clusters.
+   % /alert %
+
+   ```
+   apiVersion: anywhere.eks.amazonaws.com/v1alpha1
```

Comment:
This is a really big example. Is there a way we can trim it down in the doc to just the fields someone will need to modify? Or maybe a table that shows the values in a table?

Reply:
I cut out the gitops content, which makes it somewhat shorter. The reason I don't want to put it in a table is that I think it makes more sense visually this way. The reader would have to figure out which `cidrBlocks` and `name` values I'm talking about. I'll look for ways to trim it, but I think this is useful.

Reply:
I agree it's useful because a lot of the key names are repeated. I hope the comments are visually noticeable enough and people don't try to copy/paste the example

Reply:
I'm going to cut this whole example and point readers to the vsphere configuration page instead. Then I'll tell them how many nodes minimum the should use and what the optional config settings are.